BrainPortal/app/models/work_group.rb

```diff
@@ -77,20 +77,28 @@ def self.prepare_pretty_category_names(groups = [], as_user = nil)
     wgs
   end
 
-  def pretty_category_name(as_user) #:nodoc:
+  def pretty_category_name(as_user = nil) #:nodoc:
     return @_pretty_category_name if @_pretty_category_name
-    if self.invisible?
-      @_pretty_category_name = 'Invisible Project'
-    elsif self.public?
-      @_pretty_category_name = 'Public Project'
-    elsif self.users.count == 0
-      @_pretty_category_name = 'Empty Work Project'
-    elsif self.users.count > 1
-      @_pretty_category_name = 'Shared Work Project'
-    elsif as_user.present? && (self.creator_id == as_user.id || self.users.first.id == as_user.id)
-      @_pretty_category_name = 'My Work Project'
+    if @_pretty_category_name.blank?
```

Comment:
Please remove the if surrounding the code at line 82. It wasn't there in the original and it's useless.